lms/static/js/spec/groups/views/cohorts_spec.js

```diff
@@ -129,6 +132,18 @@ define(['backbone', 'jquery', 'common/js/spec_helpers/ajax_helpers', 'common/js/
                 );
             };
 
+            createMockVerifiedTrackCohortsJson = function (enabled) {
+                if (enabled) {
+                    return {
+                        enabled: true,
+                        verified_cohort_name: "Verified Track"
```

Comment:
This is a test file. The course authors will determine the name of the "verified track" cohort themselves.